### ISSUE TITLE
messagePreproc not working when `onMessage` is used without `onAnyMessage`

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2725]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.37.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.37.0...4.37.1) (2022-05-31)

- 🚑 fix #2725 [`#2725`](https://github.com/open-wa/wa-automate-nodejs/issues/2725)
- Release 4.37.1 [`3ec31be`](https://github.com/open-wa/wa-automate-nodejs/commit/3ec31befee1f9d7af19750951973443a7908a3b9)
- updated types-only package [`30e9847`](https://github.com/open-wa/wa-automate-nodejs/commit/30e98471c0d7f6776587f13d2cf368dd0e6b402a)
